core/src/main/kotlin/com/ritense/valtimo/actuator/health/ValtimoHealthAggregator.kt

```diff
@@ -0,0 +1,13 @@
+package com.ritense.valtimo.actuator.health
+
+import org.springframework.boot.actuate.health.Status
+import org.springframework.boot.actuate.health.StatusAggregator
+
+class ValtimoHealthAggregator(): StatusAggregator {
+    override fun getAggregateStatus(statuses: MutableSet<Status>): Status {
+        if(statuses.stream().anyMatch { s -> s.equals(Status.DOWN) }) return Status.DOWN
+        if(statuses.stream().anyMatch { s -> s.equals(Status.OUT_OF_SERVICE) }) return Status.OUT_OF_SERVICE
+        if(statuses.stream().anyMatch { s -> s.equals(Status.UNKNOWN) }) return Status.UNKNOWN
+        return Status.UP
```

Comment:
I would not make UP the default. This means that if we get any unexpected result the service will report status up. I would switch this around and only return UP when all statuses are UP, and make UNKNOWN the default.